### ISSUE TITLE
Issue downloading Forge, SSL Certifcate has Expired

### ISSUE THREAD

**Issue by kylejuliandev** [RicardG/minecraft-modpack-installer#1]
Hi,

Love this utility. It made installing Curseforge Modpacks super easy and intuitive. Although, I initially ran into issues while using it. Specifically issues around SSL Certificate verification. Below is the stack trace I got when the installer got to installing/downloading forge.

<details>
<summary>Stacktrace</summary>

```
Downloading forge-1.16.5-36.1.0-installer.jar
Traceback (most recent call last):
  File "C:\Python310\lib\urllib\request.py", line 1348, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "C:\Python310\lib\http\client.py", line 1282, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Python310\lib\http\client.py", line 1328, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Python310\lib\http\client.py", line 1277, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Python310\lib\http\client.py", line 1037, in _send_output
    self.send(msg)
  File "C:\Python310\lib\http\client.py", line 975, in send
    self.connect()
  File "C:\Python310\lib\http\client.py", line 1454, in connect
    self.sock = self._context.wrap_socket(self.sock,
  File "C:\Python310\lib\ssl.py", line 512, in wrap_socket
    return self.sslsocket_class._create(
  File "C:\Python310\lib\ssl.py", line 1070, in _create
    self.do_handshake()
  File "C:\Python310\lib\ssl.py", line 1341, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:997)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Git\minecraft-modpack-installer\InstallModPack.py", line 400, in <module>
    main();
  File "D:\Git\minecraft-modpack-installer\InstallModPack.py", line 163, in main
    r = urllib.request.urlopen(rec);
  File "C:\Python310\lib\urllib\request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python310\lib\urllib\request.py", line 519, in open
    response = self._open(req, data)
  File "C:\Python310\lib\urllib\request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "C:\Python310\lib\urllib\request.py", line 496, in _call_chain
    result = func(*args)
  File "C:\Python310\lib\urllib\request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "C:\Python310\lib\urllib\request.py", line 1351, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:997)>
```

</details>

I'm not super familiar with Python, however, I was able to bypass the SSL verification using the following snippet

```python
import ssl;

ssl._create_default_https_context = ssl._create_unverified_context
```

Many thanks!

**Comment by RicardG** [comment]
Hi,

I am surprised anyone other than me is using this little script but I am glad you like it! From the stack trace it seems like while retrieving the forge installer it failed the SSL certificate check due to an expired cert and then crashed.

I can not fix the first issue, that is up to forge to fix by updating their cert with a new valid one. If you keep getting the same issue then using your workaround will work until they update it, although, I am not recommending it for security reasons. If you could let me know what version of forge it was trying to install or the mod pack you were trying to install I can check to confirm this. But, given that the stack trace says "certificate verify failed: certificate has expired" I am pretty sure they just forgot to update it.

As for the second issue, the crash, seems like I did not put an exception handler for SSL errors. Looks like I assumed forge will always have a valid cert. Once I add some exception handling for SSL errors I will consider this issue closed.

**Comment by RicardG** [comment]
Fixed in ededf64e62ff6006dc2fdc4a788510f0a6f2b83d
